src/Controllers/DatabaseController.js

```diff
@@ -112,13 +112,14 @@ DatabaseController.prototype.untransformObject = function(
     return object;
   }
 
-  delete object.authData;
```

Comment:
Dont we want to do the same to the sessionToken? May need to check against Parse.com?